exasol_integration_test_docker_environment/lib/test_environment/parameter/external_test_environment_parameter.py

```diff
@@ -13,10 +13,12 @@ class ExternalDatabaseXMLRPCParameter(Config):
                                                               visibility=ParameterVisibility.HIDDEN)
 
 
+# should or can we use lib.test_environment.ports.PortForwarding.default_ports here?
```

Comment:
enum or constant is probably easier, but you can try

Reply:
Sorry, I was imprecise.
In file `ports.py` the default ports are (database=8888, bucketfs=6583, ssh=22)
Is there a specific reason why the current file uses a different database port (8563)?
If yes, then I'd propose to change file `ports.py` to contain different defaults, e.g. `external`, `on_premise`, etc.

My basic idea is to concentrate all information and logic about ports in a single place instead of scattering across multiple files in different directories.

Reply:
Default values below are currently unused. We will remove the defaults for all ports here.

Docker DB had default 8888 in earlier times, different from port of normal deployments.
Current bucketfs port is also different. 
This requires major version change, that should be separated from the current one. 
I will create a ticket.